src/block/accordion/edit.js

```diff
@@ -227,3 +231,41 @@ addFilter( 'stackable.block-default-styles.use-saved-style', 'stackable/icon-lab
 	}
 	return enabled
 } )
+
+// Return default icon for accordion
+addFilter( 'stackable.block-component.icon.default', 'stackable/accordion', starIcon => {
+	const { clientId } = useBlockEditContext()
+	const { getActiveBlockVariation } = useSelect( 'core/blocks' )
+
+	const { isAccordionIcon, block } = useSelect(
+		select => {
+			const { getBlock } = select( 'core/block-editor' )
+			const { parentTree } = select( 'stackable/block-context' ).getBlockContext( clientId )
+			const columnClientId = nth( parentTree, -2 )?.clientId
+			const accordionClientId = nth( parentTree, -3 )?.clientId
+			const iconLabelName = nth( parentTree, -1 )?.name
+			const columnName = nth( parentTree, -2 )?.name
+			const accordionName = nth( parentTree, -3 )?.name
+			if ( ! iconLabelName || ! columnName || ! accordionName ) {
+				return false
+			}
+			if ( iconLabelName !== 'stackable/icon-label' ||
+			     columnName !== 'stackable/column' ||
+				 accordionName !== 'stackable/accordion' ) {
+				return false
+			}
+			return {
+				isAccordionIcon: ! getBlock( accordionClientId ).innerBlocks[ 0 ].clientId !== columnClientId,
+				block: getBlock( accordionClientId ),
+			}
+		},
+		[ clientId ]
+	)
+
+	if ( isAccordionIcon ) {
+		const activeVariation = getActiveBlockVariation( block?.name, block?.attributes )
+		return ( activeVariation.name === 'plus' )
+			? applyFilters( 'stackable.block-component.plus.icon-open' ) : defaultIcon
+	}
+	return starIcon
+} )
```

Comment:
You may find the `parentTree` provided by the `useBlockContext` hook helpful here